docs/gathered-data.md

```diff
@@ -69,6 +69,34 @@ None
 None
 
 
+## AggregatedInstances
```

Comment:
I think the name is a bit too general. What about something like `PrometheusInstanceNames` ?

Reply:
It also gets Alertmanager instances, but `AlertmanagerPrometheusInstanceNames` sounds a bit lengthy.
Anyway no strong opinions.